include/networkit/generators/quadtree/QuadNodeCartesianEuclid.hpp

```diff
@@ -429,7 +429,12 @@ class QuadNodeCartesianEuclid {
 			}
 		}
 		count finalNeighbours = result.size();
-		if (probLB == 1) assert(finalNeighbours == incomingNeighbours + ownsize);
+		if (probLB == 1){
+			assert(finalNeighbours == incomingNeighbours + ownsize);
+			(void)finalNeighbours;
```

Comment:
we have `tlx::unused` for this -- it's much more verbose

Reply:
I won't fight over it, but the first time I came across this construct, it took me a few moments to understand what's going on. And using `tlx::unused` clearly states intent.